github_repositories/variables.tf

```diff
@@ -3,3 +3,9 @@ variable "repository_name" {}
 variable "secrets" {
   default = {}
 }
+
+variable "collaborators" {
+  description = "Outside collaborators to add to the repository"
```

Comment:
Maybe make the description clearer, so that its specific its users who are not part of the TNA Github organisation?

Reply:
Makes sense, I've updated the description.